files/tests/test_community_impact.py

```
@@ -426,26 +534,60 @@ def test_manage_page_superuser_allowed(self, _mock_vite):
         "django_vite.core.asset_loader.DjangoViteAssetLoader.instance",
         return_value=make_vite_loader_mock(),
     )
-    def test_manage_page_advanced_user_allowed(self, _mock_vite):
+    def test_manage_page_advanced_user_redirected(self, _mock_vite):
         self.client.login(username="advancedimpact", password="testpass123")
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)